<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackWidget</class>
 <widget class="QWidget" name="TrackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>359</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSpinBox" name="nombre1">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="operation">
      <item>
       <property name="text">
        <string>+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="nombre2">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="boutonEgal">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="result">
      <property name="text">
       <string>RÃ©sultat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QVTKWidget" name="visualizationBox" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="lines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>lines</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="tubes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tubes</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="totalTimeRange">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>55</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TimeRange</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GlyphTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TrackTimeRange</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="begin">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>55</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="end">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>55</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="glyphTime">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>55</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply parameters</string>
   </property>
  </widget>
  <widget class="QComboBox" name="glyphShape">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>sphere</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cube</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="snapshot">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Take Snapshot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
